reference/api/state_api.md

```diff
@@ -274,6 +275,79 @@ None.
 curl -X "DELETE" http://localhost:3500/v1.0/state/starwars/planet -H "ETag: xxxxxxx"
 ```
 
+## State transactions
+
+Persists the changed to the state for an actor as a multi-item transaction.
```

Comment:
"for an actor" - this is not just for actors anymore

Reply:
removed :)